cypress/integration/integration_spec.js

```diff
@@ -15,9 +15,26 @@ function acceptCookieConsent() {
 
 it('should test common interactions', () => {
     cy.visit('/');
-
     cy.checkA11y();
 
+    // Switch to welsh language version of homepage
+    cy.get('.qa-lang-switcher').click();
+
+    // Check welsh language text
+    // When people are in the lead, communities thrive
+    cy.findByText('Rydym yn cefnogi pobl a chymunedau i ffynnu').should(
+        'be.visible'
+    );
+
+    // Switch back to English
+    cy.get('.qa-lang-switcher').click();
+
+    // Check welsh language text
+    // When people are in the lead, communities thrive
```

Comment:
Quite like this just being a comment in the source but probably don't need it twice?